### DISCUSSION TITLE
Unable to start the watcher using the CLI

### DISCUSSION THREAD

**Discussion by JonathanMMunoz** [tailwindlabs/tailwindcss#8750] [Help]
On my Mac I have no issue starting the watcher. When pulling the exact same project on my Windows laptop I am unable to fire up the watcher and get the following error:

```
Rebuilding...
CssSyntaxError: tailwindcss: D:\Projecten\shop\wp-content\themes\myTheme\static\css\tailwind.css:16:1: 'spacing[some key][1.5]' does not exist in your theme config. 'spacing' has the following keys: '0', '2', '4', '6', '8', '10', '12', '14', '15', '20', '25', '30', '35', '40', '43', '45', '50', '60', '65', '80', '100', '110', '120', '200', '240', '300', '400', '500', '600', '700', '10p', '20p', '25p', '30p', '40p', '50p', '60p', '70p', '75p', '80p', '90p', '100p'

  14 | @tailwind base;
  15 | @tailwind components;
> 16 | @tailwind utilities;
     | ^
  17 |
  18 | @layer base {
```

I read that this is often due to the 'content' in the config file not being set up accordingly, but it works on one machine and not on the other.

`content: ["./**/*.{php,js,jsx}"],`

Already updated NodeJS but it does not solve the issue.

Any ideas anyone?

**Comment by puerschel93** [comment]
Same issue here

**Comment by tofuwu-dev** [comment]
Hi @JonathanMMunoz 
Are you sure that you have configured properly tailwind on your project?

**Comment by almenjonatan** [comment]
@JonathanMMunoz 

I just had this issue, apparently you cannot have content the glob pattern at root level. I had to be more specific.
